nest-cli/src/commands/pull.rs

```diff
@@ -0,0 +1,50 @@
+use failure::{format_err, Error, ResultExt};
+use indicatif::{ProgressBar, ProgressStyle};
+use libnest::config::Config;
+use libnest::transaction::{PullTransaction, Transaction};
+
+use super::download::Download;
+use super::{ask_confirmation, print_transactions};
+
+pub fn pull(config: &Config) -> Result<(), Error> {
+    let transactions: Vec<_> = config
+        .repositories()
+        .into_iter()
+        .map(|repository| Transaction::Pull(PullTransaction::from(repository)))
+        .collect();
+
```

Comment:
What if there is no transaction? Maybe a message saying "No repository to pull, quitting." (or something similar) would be neat, nah?